packages/core/src/lib/server.ts

```diff
@@ -98,13 +103,13 @@ export const enableHealthCheck = (
         return Promise.resolve();
       },
     },
-    logger: (message: string, error: Error): void => {
-      logger.error(`HEALTH CHECK ERROR ${message}`, error, error.stack);
-    },
-  };
+  } : {};
 
-  app.listen = (...args: any) => server.listen.apply(server, args);
-  createTerminus(server, terminusOptions);
+  return {
+    ...defaultTerminusOptions,
+    ...terminusOptions,
+    ...healthcheckOptions,
```

Comment:
Does this mean that health check will overwrite terminusOptions when the same `/healthCheck` path gets passed along? (similar Q to what I asked in the examples

Reply:
That s correct; they do the same thing

Reply:
The moment you pass the `healthcheck:enabled`, you lose the ability to pass the healthcehck via terminus.